allComponents.css

```diff
@@ -8,3 +8,4 @@
 @import url(./pages/components/textUtilities/textUtil.css);
 @import url(./pages/components/modal/modal.css);
 @import url(./pages/components/toast/toast.css);
```

Comment:
For Toast if you click on the close button its size increases.
Else everything looks good!
Good to merge!